apps/admin_app/assets/js/views/product/edit.js

```diff
@@ -31,6 +32,25 @@ export function imageOnEnter(){
   });
 }
 
+export function defaultImage(){
+  $(document).delegate('input[type="checkbox"]', 'click', function(event) {
+    $(this).parent().siblings().find(':checkbox').prop('checked', false);
+    var default_image = $(this).val();
+    var product_id = $("#product-id").val();
+    var data = { product_id: product_id,  default_image: default_image};
+    var CSRF_TOKEN = $("meta[name='csrf-token']").attr("content");
+    $.ajax({
+      url: `/set-default-image/${product_id}`,
+      type: "POST",
+      cache: false,
+      data: data,
+      beforeSend: function (xhr) {
+        xhr.setRequestHeader("X-CSRF-Token", CSRF_TOKEN);
+      }
+    });
+ });
+}
+
 export function imageOnSubmit(){
```

Comment:
@jyotigautam The function name is clear to describe the working of the function. This can be better.

Reply:
imageOnSubmit function is not required now since we have removed the submit button from the upload image page.